docker-compose.yml

```
@@ -19,6 +19,8 @@ services:
     build: .
     # This uses the value of the LAMBDA_TAG environment variable from
     # the invoking environment but falls back to a default value.
-    image: cisagov/example_lambda:${LAMBDA_TAG:-latest}
+    image: cisagov/nvdsync_lambda:${LAMBDA_TAG:-latest}
+    environment:
+      - AWS_LAMBDA_FUNCTION_TIMEOUT=900000
```

It never hurts to add a comment like this:
```suggestion
      # 900,000 milliseconds = 15 minutes
      - AWS_LAMBDA_FUNCTION_TIMEOUT=900000
```

---

@dav3r I addressed this here: https://github.com/cisagov/cyhy-nvdsync-lambda/pull/2/commits/5764f7c19a4cb425074d410a44b1f8ec36ef858d

---

@michaelsaki - Why are you averse to simply committing the suggestion via @dav3r's comment?

---

> @michaelsaki - Why are you averse to simply committing the suggestion via @dav3r's comment?

@jsf9k - I wasn't aware that you could simply commit them through the suggestion. Thank you for pointing out the capability in GitHub. Will definitely be using it in the future!